src/main/java/org/eclipse/tractusx/ssi/lib/did/web/DidWebResolver.java

```
@@ -83,9 +83,12 @@ public DidDocument resolve(Did did) throws DidResolverException {
       final Map<String, Object> json = mapper.readValue(body, Map.class);
 
       return new DidDocument(json);
+    } catch (InterruptedException e) {
+      Thread.currentThread().interrupt();
```

Please throw an exception instead of returning null